source/connect/config-file.txt

```diff
@@ -0,0 +1,62 @@
+.. _config-file:
+
+==========================
+Compass Configuration File 
+==========================
+
+You can configure |compass| settings using a :manual:`configuration file 
+</reference/configuration-options>`. Configure settings with a configuration 
+file to help manage the command line options for your large-scale deployments. 
+
+Use Cases
+---------
+
+If you do not want your settings modified after startup, the configuration file 
```

Comment:
[suggestion]

The fact that settings set in the config file can't be modified after startup might be good to mention in the Behavior section?

Reply:
Agreed, done!